src/Codeception/Step.php

```diff
@@ -106,6 +107,9 @@ protected function getArgumentsAsString(array $arguments)
 
     protected function parseArgumentAsString($argument)
     {
+        if (class_exists('\Facebook\WebDriver\WebDriverBy') && $argument instanceof \Facebook\WebDriver\WebDriverBy) {
```

Comment:
For what it's worth, I followed the same paradigm as https://github.com/Codeception/Codeception/blob/4eb7ccf2f1369998fc3bfdeed896ac4e76f7148a/src/Codeception/Util/Locator.php#L195, but I'll change it if you prefer.